algorithmic_efficiency/workloads/wmt/wmt_pytorch/workload.py

```diff
@@ -43,8 +44,8 @@ def compute_weighted_cross_entropy(self,
       Tuple of loss for every example and batch normalizing factor.
```

Comment:
This docstring is no longer accurate, right? It should just be per-example losses?

Reply:
Nice catch, fixed.